firmware/user/utils/esp_niceness.h

```diff
@@ -6,6 +6,9 @@ void rom_i2c_writeReg_Mask(uint8_t block, uint8_t host_id, uint8_t reg_add, uint
 void read_sar_dout(uint16_t* );
 uint32_t xthal_get_ccount(void);
 char* ets_strcat(char* dest, const char* src);
+uint32_t system_get_rtc_time(void); //XXX TESTME (note, probably inaccurate, will also need system_rtc_clock_cali_proc) "If system_get_rtc_time returns 10 (which means 10 RTC cycles), and system_rtc_clock_cali_proc returns 5.75 (which means 5.75 μs per RTC cycle), the real time is 10 x 5.75 = 57.5 μs.
```

Comment:
Two out of three of these prototypes already exist in `<user_interface.h>`, so please remove them from `"esp_niceness.h"` and use the header from espressif instead.
```
uint32 system_get_rtc_time(void);
uint32 system_get_time(void);
```
I'm not seeing `hw_timer_get_count_data()` in a library file either. That may be an RTOS specific function?
https://docs.espressif.com/projects/esp8266-rtos-sdk/en/latest/api-reference/peripherals/hw_timer.html

Reply:
yeah, I tried compiling a call to `hw_timer_get_count_data()` and it fails. So you should revert the changes to this file.
```
user_main.c:(.irom0.text+0xe): undefined reference to `hw_timer_get_count_data'
```